config/examples/awesim/initializers/ood.rb

```diff
@@ -11,8 +11,8 @@
 end
 
 # don't show develop dropdown unless you are setup for app sharing
-Configuration.app_development_enabled = UsrRouter.base_path.directory?
-Configuration.app_sharing_facls_enabled = true
+AppConfig.app_development_enabled = UsrRouter.base_path.directory?
```

Comment:
This is a breaking change. Are there any OOD partners that use a custom initializer that makes calls to `Configuration`? If not, then this should be fine.

Reply:
Now using `Configuration` for the singleton instance like before.